### ISSUE TITLE
Options to modify SAN domains on an existing managed cert would be nice

### ISSUE THREAD

**Issue by dtucny** [Neilpang/acme.sh#839]
# Background
Issuing a new cert can lead to a quite long command line, especially once you've added custom file locations, verification details and hooks.
Conveniently, all this is then saved in the .conf file so that renewals are painless, automatic, or if you want to manually renew, a simple `acme.sh -r -d <domain>` will take care of everything.
In the event you want to add an additional name to an existing cert, it would appear that the current preferred approach is to run the issue command again with the additional `-d <name>` option, which means remembering what you entered on the previous issue command or trying to build the command again based on the contents of the .conf file.
It is reasonably straight forward to modify the .conf file manually and add/remove names from 'Le_Alt', then forcing a renew, which takes care of everything nicely.

# Feature Request
It would be nice if it was possible from the script to be able to modify the Alt/SAN domains list, triggering a reissue at the same time.

Potentially implemented as something along the lines of the following:
--modify         used with -d allows modification of an already issued certs options, e.g. to add a hook, change paths, modify renew command or to modify alt names
-D domain.tld valid with --modify to remove an alt name

e.g.
```
# acme.sh --list
Main_Domain     KeyLength  SAN_Domains                                     Created                       Renew
example.com     ""         www.example.com,test.example.com                Fri 12 May 04:01:06 UTC 2017  Tue 11 Jul 04:01:05 UTC 2017

# acme.sh --modify -d example.com -d mail.example.com -D test.example.com
... Success
	
# acme.sh --list
Main_Domain     KeyLength  SAN_Domains                                     Created                       Renew
example.com     ""         www.example.com,mail.example.com                Fri 12 May 04:05:06 UTC 2017  Tue 11 Jul 04:05:05 UTC 2017

```

**Comment by Neilpang** [comment]
Yes,  I would like to add new commands like `--add-domain` or so.
The expected procedure would look like:

```sh
acme.sh --issue  -d example.com  -d sub1.example.com

acme.sh --add-domain  -d example.com  -d  sub2.example.com

acme.sh --renew -d example.com

```

**Comment by chinkung** [comment]
It would be great if you could implement **--remove-domain** as well as sometime I need to manually remove unused sub domain from **root_domain.conf** myself.

**Comment by bastischubert** [comment]
any news on this feature? would be awesome to see it in one of the next releases ;)

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
